packages/vite/src/node/plugins/css.ts

```diff
@@ -674,7 +691,18 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
       await urlEmitQueue.run(async () =>
         Promise.all(
           urlEmitTasks.map(async (info) => {
-            info.content = await finalizeCss(info.content, true, config)
+            info.content = await finalizeCss(
+              info.content,
+              true,
+              config,
+              info.originalFilename,
+              [
+                {
+                  file: info.originalFilename,
+                  end: getLineCount(info.content),
+                },
+              ],
```

Comment:
```suggestion
```
When the input is not concatenated, could we skip passing this argument?

Reply:
I changed the type of the `concatCssEndLines` to be optional.